content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -8,6 +8,8 @@ weight: 140
 
 This page shows how to configure liveness, readiness and startup probes for containers.
 
+You may wants to have an overview of [Liveness, Readiness and Startup Probes](/docs/concepts/configuration/liveness-readiness-startup-probes)
```

Comment:
Can you reword the sentence to something like:
For an overview about probes, see [Liveness, Readiness, and Startup Probes](/docs/concepts/configuration/liveness-readiness-startup-probes).